cms/static/js/spec/views/paging_spec.js

```diff
@@ -380,193 +384,5 @@ define([ "jquery", "common/js/spec_helpers/ajax_helpers", "URI",
                     });
                 });
             });
-
-            describe("PagingFooter", function () {
```

Comment:
That's a very good point, @dan-f. Now that it is up to each view to include its own footer, these tests should belong in the spec for the view. I looked and the existing asset_spec.js doesn't have any footer tests, so can you add a minimal set that verifies that the basic behavior works. I also noticed that the asset_spec has unused variables for its templates that I forgot to remove when I switched the code over to RequireJS Text. Could you fix that at the same time, please.